cognite/client/data_classes/relationships.py

```diff
@@ -78,47 +78,59 @@ def _resolve_target(target):
 
 # GenClass: relationshipsAdvancedListRequest.filter
 class RelationshipFilter(CogniteFilter):
-    """Filter on relationshisp with exact match.
+    """Filter on relationships with exact match. Multiple filter elments in one property, e.g. `dataSets: [ "a", "b" ]`, will return all relationships that are either in dataSet `a` or in dataSet `b`. Filters in multiple properties will return the relationships that match all criteria. Filters on a `resourceId` without a `resource` (type) in sources and targets will return relationships that match the resourceId and match any resource type.  
```

Comment:
arguably "...will return all relationships where the dataSet field is either `a` or `b`"

Reply:
although this field needs to die in favor of proper dataSet support

Reply:
these are auto generated from your specs :)